src/__tests__/extensions/toolbar.js

```diff
@@ -158,4 +158,24 @@ describe('Toolbar', () => {
             expect(given.subject()).toBe(false)
         })
     })
+
+    describe('afterDecideResponse', () => {
+        given('localStorage', () => ({
+            setItem: jest.fn(),
+            removeItem: jest.fn(),
+        }))
+
+        it('should enable the toolbar based on the /decide response', () => {
+            given('decideResponse', () => ({ isAuthenticated: true, editorParams: { toolbarVersion: 'toolbar' } }))
+            given.toolbar.afterDecideResponse(given.decideResponse, given.localStorage)
+            expect(given.localStorage.setItem).not.toHaveBeenCalled()
+            expect(given.localStorage.removeItem).toHaveBeenCalledWith('toolbar_disabled')
+        })
+
+        it('should disable the toolbar based on the /decide response', () => {
```

Comment:
Might also want to add a test to actually make sure the toolbar is not initialized when the flag is set in localStorage?